hastory-cli/src/Hastory/Cli/OptParse.hs

```diff
@@ -82,7 +82,7 @@ getArguments = do
   handleParseResult result
 
 getEnvironment :: IO Environment
-getEnvironment = Env.parse (Env.header "hastory") (Env.prefixed "HASTORY_" envParser)
+getEnvironment = Env.parse (Env.header "hastory") envParser
```

Comment:
Good, but let's also show the env vars in the `--help` screen so we can check (and for docs purposes).